### PR TITLE
Feedback tests

### PR DESCRIPTION
**Pull request by eledavi**
Created test for getReinforcement()

### CODE REVIEW

File: client/domain/FeedbackObjectFactorySpec.js [hunk 1]
```diff
@@ -50,5 +50,7 @@ describe('FeedbackObjectFactory', function() {
       expect(feedback.getParagraphs.length).toEqual(0);
     });
   });
+
```
Comment (seanlip) [inline client/domain/FeedbackObjectFactorySpec.js]: Why is this file being modified?

File: client/domain/SnapshotObjectFactorySpec.js [hunk 2]
```diff
@@ -30,6 +30,13 @@ describe('SnapshotObjectFactory', function() {
     });
   }));
 
+  describe('setPrereqCheckFailure', function() {
+    it('should correctly set and get prereqCheckFailure', function() {
+      snapshot.setPrereqCheckFailure("true");
```
Comment (seanlip) [inline client/domain/SnapshotObjectFactorySpec.js]: Is this an actual possible value for PrereqCheckFailure that can be achieved in practice?

**Comment by seanlip** [comment]
Please merge master into your branch. See notes in #340; there should be instructions in the playbook to do this.

File: client/domain/SnapshotObjectFactorySpec.js (hunk 2 above)
Comment (eledavi) [inline client/domain/SnapshotObjectFactorySpec.js]: Yes, I changed that in the Snapshot tests PR (#330).

File: client/domain/FeedbackObjectFactorySpec.js (hunk 1 above)
Comment (eledavi) [inline client/domain/FeedbackObjectFactorySpec.js]: I'm not sure why it's doing this but I reverted this file to a previous commit so it should be good now.

Comment (seanlip) [inline client/domain/FeedbackObjectFactorySpec.js]: It's not. Please see https://github.com/google/tie/pull/339/files

**Comment by seanlip** [comment]
NB: You might need to resolve the merge conflicts here that resulted from the other test PRs being merged in.

**Comment by eledavi** [comment]
@seanlip I merged master into this branch and resolved the conflicts. It should be good now.

**Comment by seanlip** [comment]
It's not. Please look at the "Files Changed" tab and address all remaining comments.

**Comment by eledavi** [comment]
@seanlip Done

**Pull request merged**